src/ApiService/ApiService/OneFuzzTypes/Requests.cs

```diff
@@ -192,57 +211,68 @@ public record ScalesetSearch(
     Guid? ScalesetId = null,
     List<ScalesetState>? State = null,
     bool IncludeAuth = false
-);
+) : BaseRequest;
 
 public record ScalesetStop(
     Guid ScalesetId,
     bool Now
-);
+) : BaseRequest;
 
 public record ScalesetUpdate(
     Guid ScalesetId,
     [property: Range(1, long.MaxValue)]
     long? Size
-);
+) : BaseRequest;
 
-public record TaskGet(Guid TaskId);
+public record TaskGet(Guid TaskId) : BaseRequest;
+
+public record TaskCreate(
+   Guid JobId,
+   List<Guid>? PrereqTasks,
+   TaskDetails Task,
+   [property: Required]
```

Comment:
in ASP.NET Core they make `[Required]` the default for any non-null properties, we should try to do the same